### ISSUE TITLE
confused about the BN layer

### ISSUE THREAD

**Issue by iSmida** [jwyang/fpn.pytorch#7]
Hi,thanks for your works.
The BN layer in pytorch is  
class torch.nn.BatchNorm2d(num_features, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
in your code you choose the default parameters.  Is that right for object detection with a small batchsize?
